### PR TITLE
New Update functional-api-guide.md

### PR DESCRIPTION
**Pull request by PassionateLooker**
### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [ ] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Review by danik292 — APPROVED** [review]

File: docs/templates/getting-started/functional-api-guide.md [hunk 1]
```diff
@@ -156,9 +156,7 @@ model.compile(optimizer='rmsprop',
               loss={'main_output': 'binary_crossentropy', 'aux_output': 'binary_crossentropy'},
               loss_weights={'main_output': 1., 'aux_output': 0.2})
 
-# And trained it via:
-model.fit({'main_input': headline_data, 'aux_input': additional_data},
-          {'main_output': headline_labels, 'aux_output': additional_labels},
+
           epochs=50, batch_size=32)
```
Comment (danik292) [inline docs/templates/getting-started/functional-api-guide.md:160]: This goes out😉.

**Comment by danik292** [comment]
@PassionateLooker Plese look at the coments. 😑

**Comment by gbaned** [comment]
Hi @PassionateLooker Can you please check @danik292's comments and resolve conflicts?. Thank you!

File: docs/templates/getting-started/functional-api-guide.md [hunk 2]
```diff
@@ -156,9 +156,7 @@ model.compile(optimizer='rmsprop',
               loss={'main_output': 'binary_crossentropy', 'aux_output': 'binary_crossentropy'},
               loss_weights={'main_output': 1., 'aux_output': 0.2})
 
-# And trained it via:
-model.fit({'main_input': headline_data, 'aux_input': additional_data},
-          {'main_output': headline_labels, 'aux_output': additional_labels},
```
Comment (haifeng-jin) [inline docs/templates/getting-started/functional-api-guide.md:161]: Why are these lines removed?

**Review by haifeng-jin — COMMENTED** [review]
Thanks for the PR! Please also resolve the conflicts

**Comment by gbaned** [comment]
Hi @PassionateLooker Can you please check @haifeng-jin's comments and keep us posted ? Thank you!
